src/gui/transferlistwidget.cpp

```diff
@@ -871,7 +871,7 @@ void TransferListWidget::displayListMenu(const QPoint &)
     auto *actionFirstLastPiecePrio = new TriStateAction(tr("Download first and last pieces first"), listMenu);
     connect(actionFirstLastPiecePrio, &QAction::triggered, this, &TransferListWidget::setSelectedFirstLastPiecePrio);
     auto *actionAutoTMM = new TriStateAction(tr("Automatic Torrent Management"), listMenu);
-    actionAutoTMM->setToolTip(tr("Automatic mode means that various torrent properties(eg save path) will be decided by the associated category"));
+    actionAutoTMM->setToolTip(tr("Automatic mode means that various torrent properties (e.g. save path) will be decided by the associated category"));
```

Comment:
If you make any other changes in the PR, could you undo this? I'm moving it elsewhere in my PR and i can fix it there.
Otherwise i'll just fix the conflict, no problem.